### DISCUSSION TITLE
benchmarking

### DISCUSSION THREAD

**Discussion by electronstudio** [electronstudio/raylib-python-cffi#126] [Announcements]
I've been doing some benchmarking of drawing textures https://github.com/electronstudio/megabunny/

The performance of pypy and graalpy python implementations is now very impressive.

**Comment by Hammer2900** [comment]
You should describe how much performance has increased, it's just a short note )))

**Comment by electronstudio** [comment]
On that benchmark, which only really tests FFI speed:

Version | mbunnies
--- | ---
C | 9.6
GraalPy | 5.9
Pypy 3.10 | 5.1
CPython 3.10 | 3.4
CPython 3.12 | 3.1

I think the differences would be even greater if you also benchmarked game logic

**Comment by electronstudio** [comment]
Bunnymark including logic:

Version | mbunnies
--- | ---
CPython 3.12| 2.5
CPython 3.10 | 2.3
PyPy 3.10| 16.5
GraalPy 3.10 | 26.7
